IdentityCore/src/validation/MSIDB2CAuthority.m

```diff
@@ -28,11 +28,12 @@
 
 @implementation MSIDB2CAuthority
 
-- (instancetype)initWithURL:(NSURL *)url
-                    context:(id<MSIDRequestContext>)context
-                      error:(NSError **)error
+- (nullable instancetype)initWithURL:(NSURL *)url
+                      validateFormat:(BOOL)validateFormat
+                             context:(id<MSIDRequestContext>)context
+                               error:(NSError **)error
 {
-    self = [super initWithURL:url context:context error:error];
+    self = [super initWithURL:url validateFormat:validateFormat context:context error:error];
     if (self)
     {
         _url = [self.class normalizedAuthorityUrl:url context:context error:error];
```

Comment:
This method will return nil if the format is invalid. I assume if validation is 'NO', we shouldn't normalize url, maybe just spaces trimming.

Reply:
I'd argue that we should remove the authorityformat check in normalizing.